tests/parser/TuningTests.cpp

```diff
@@ -223,8 +227,10 @@ BOOST_AUTO_TEST_CASE(TuningTest) {
       const auto& tuning = schedule[timeStep].tuning();
 
       BOOST_CHECK(event.hasEvent(ScheduleEvents::TUNING_CHANGE));
-      double TSINIT = tuning.TSINIT;
-      BOOST_CHECK_CLOSE(TSINIT, 2 * Metric::Time, diff);
+			std::optional<double> TSINIT = tuning.TSINIT;
+      BOOST_CHECK(TSINIT.has_value());
+      //BOOST_CHECK_CLOSE(TSINIT.value(), 2 * Metric::Time, diff);
```

Comment:
Is this intentional?  Are we no longer checking that `TSINIT` has the expected value, just that there is a value?

Reply:
Not intentional, at least I cannot now recall a good reason why. Will fix.